src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,44 @@
+package duke.task;
+
+/**
+ * The abstract class for Task.
+ * @author Toh Li Yuan (A0255811H)
+ */
+public abstract class Task {
+    protected static final String DISCRIMINATOR = " || ";
+    protected String name;
+    protected boolean status;
```

Comment:
Perhaps you could use a name to make the status sound more like a boolean.

Reply:
Good point, noted for future revisions